### ISSUE TITLE
Does not decompile inner jar files

### ISSUE THREAD

**Issue by reshampanth** [kwart/jd-cmd#38]
I have a ear file with multiple jar files inside it. On using this tool, only the class files inside are decompiled, files inside inner jars are untouched

**Comment by kwart** [comment]
Yes, you're right. Archives are standard resources from the jd-cli point of view. If you have an idea of how to improve the behavior, feel free to send a pull request with the improvement.
